src/styles/StyleUtils.js

```diff
@@ -605,13 +605,24 @@ function hasSafeAreas(windowWidth, windowHeight) {
 }
 
 /**
- * Get variable keyboard height as style
- * @param {Number} keyboardHeight
+ * Get height as style
+ * @param {Number} height
  * @returns {Object}
  */
-function getHeight(keyboardHeight) {
+function getHeight(height) {
     return {
-        height: keyboardHeight,
+        height,
+    };
+}
+
+/**
+ * Get height as style
+ * @param {Number} minHeight
+ * @returns {Object}
+ */
+function getMinHeight(minHeight) {
```

Comment:
Let's use full function and variable names in `getMinimumHeight` and `minimumHeight`